README.md

```diff
@@ -8,59 +8,52 @@ The Bubble Sort algorithm is implemented as a method in a straightforward manner
 ## Built With
 
 - Ruby 3.0.1
-- Gem, bundler, RuboCop
+- Gems:
+  - Bundler
+  - RuboCop
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+To get a local copy up and running follow these simple steps.
 
-
-To get a local copy up and running follow these simple example steps.
+On GitHub bash
+`git clone https://github.com/notlfish/ruby-bubble-sort.git`
```

Comment:
After cloning what?

You can mention the command to enter into your folder before running your ruby file or rubocop.
` $ cd ruby-bubble-sort`